src/shell/shell_misc.cpp

```diff
@@ -32,6 +32,31 @@
 #include "string_utils.h"
 #include "../ints/int10.h"
 
+namespace {
+class Int10Manager {
```

Comment:
Can you pick a better name that more accurately reflects what this class is doing? Just by looking at the name, it's reasonable to assume it "manages" all BIOS interrupt 10h functionality. Clearly, that's not the case here; it only deals with a rather small subset related to console I/O.